src/pages/LoginPage.jsx

```diff
@@ -0,0 +1,12 @@
+import Nav from "../components/Nav";
+import LoginForm from "../components/LoginForm";
+
+export default function LoginPage(){
+    return (<>
+        <Nav/>
```

Comment:
use a good code formatter like Prettier it makes your code more presentable. For eg: 
![prettier](https://user-images.githubusercontent.com/93508491/161200486-2fa7ab1e-ada5-485a-b453-01d8a4238569.JPG)
.